OrbitGl/DataManager.cpp

```diff
@@ -60,11 +60,21 @@ void DataManager::set_visible_functions(absl::flat_hash_set<uint64_t> visible_fu
   visible_functions_ = std::move(visible_functions);
 }
 
+void DataManager::set_selected_thread_id(ThreadID thread_id) {
```

Comment:
drop ThreadID? use int32_t instead? we are using it elsewhere for pid for example.

Reply:
done.